src/Compilers/VisualBasic/Portable/VBResources.resx

```diff
@@ -1995,6 +1995,9 @@
   <data name="ERR_TypeRefResolutionError3" xml:space="preserve">
     <value>Import of type '{0}' from assembly or module '{1}' failed.</value>
   </data>
+  <data name="ERR_ValueTupleTypeRefResolutionError" xml:space="preserve">
+    <value>Import of type '{0}' failed.</value>
```

Comment:
>Import of type '{0}' failed. [](start = 11, length = 28)

The message feels somewhat misleading, there was no actual import failure. Consider using something like "Unable to locate type {0}."